static/src/stylesheets/module/commercial/glabs/_hosted.scss

```diff
@@ -642,3 +644,30 @@ $hosted-stripe: 28px;
 .hosted__popup-text {
     font-weight: 200;
 }
+
+.hosted__central-play-button {
+    @include f-headlineSans;
```

Comment:
My first thought was to create a mixin in `_mixins.scss`. There is already one there named `absolute-center`, but it seems more a mixin to _cover_ a container in an element, rather than center it.